api/data_refinery_api/tests.py

```diff
@@ -342,320 +328,42 @@ def test_compendia(self):
         drc1.s3_key = "drc2.tsv"
         drc1.save()
 
-        response = self.client.get(reverse('compendia'))
-        response_json = response.json()
+        response = self.client.get(reverse('computed-files'), {'is_compendia': True})
+        response_json = response.json()['results']
         self.assertEqual(3, len(response_json))
         # Prove that the download_url field is missing and not None.
-        self.assertEqual('NotPresent', response_json[0].get('download_url', 'NotPresent'))
+        self.assertEqual(None, response_json[0].get('download_url', None))
```

Comment:
Why did you make this change?

Reply:
good catch! I'll update the serializers and restore this.